service-onboarding-build-pack/Jenkinsfile

```diff
@@ -460,7 +459,7 @@ def setRepoUrl(repo_protocol, scm_repo_name, repo_base)
 	return repo_url
 }
 
-def createProjectInSCM(gitlab_private_token, gitlab_cas_name_space_id, repo_base, scm_repo_name, username, password, cas_rest_repo)
+def createProjectInSCM(scm, gitlab_private_token, gitlab_cas_name_space_id, repo_base, scm_repo_name, username, password, cas_rest_repo)
```

Comment:
The signature for this method looks wrong. Why would createProjectinSCM take a specific SCM's parameter (gitlab_private_token)?

Reply:
I believe Daimen's story will fix this issue where we have scmmodule